docs/env-variables.md

```diff
@@ -0,0 +1,23 @@
+# Different Environment Variables used
+
+This page describes about the different environment variables that are
+used to configure kube-scheduler-simulator.
+
+`Port`: This is the port number on which kube-scheduler-simulator is started and it's value is 1212 which is set [here](https://github.com/kubernetes-sigs/kube-scheduler-simulator/blob/3111ec419ccb8c49197b75385fbd166f7f159435/docker-compose.yml#L7)
+
+`EtcdURL`: This is the URL for kube-scheduler-simulator etcd which runs locally and it's value is set [here](https://github.com/kubernetes-sigs/kube-scheduler-simulator/blob/3111ec419ccb8c49197b75385fbd166f7f159435/docker-compose.yml#L8)
+
+`FrontendURL`: This URL is on which kube-scheduler-simulator is reachable locally and it's value is set [here](https://github.com/kubernetes-sigs/kube-scheduler-simulator/blob/3111ec419ccb8c49197b75385fbd166f7f159435/docker-compose.yml#L9).
+
+`KUBECONFIG`: This variable is used to access the cluster which is
+imported locally. [Read
```

Comment:
@pravarag thanks!
hmm, at least, in our simulator, this value is used to access the cluster when we'd like to import the cluster's resources. So, I think that it is not a `locally imported cluster` but a `cluster which will be imported locally`.
(But my English is not good, sorry...)
WDOT?

Reply:
How about, 

```
This variable is used by kubectl tool to find the information it needs to access the cluster which will be imported locally and communicate with the API server of the cluster. 
```
I took some reference from the official Kubernetes docs.

Reply:
Let me know your thoughts on the above

Reply:
`KUBECONFIG`
[In general simulator]: Currently not used.
[In Beta]: Used for settings indicating clusters to be imported. To be precise, the KubeConfig file of the cluster to be imported is specified so that simulator can read the resources in the specified cluster.

So, How is this?
```
`KUBECONFIG`: Currently not used. [Beta: Existing Cluster Importing] This variable is used to find the information it needs to access the cluster which will be imported to our simulator and communicate with the API server of the cluster. If you use the Beta feature, you need to set the `Kubeconfig` file path of the cluster which will be imported. [Read more]
```

Reply:
Updated.